libs/langchain/tests/unit_tests/chat_models/test_anthropic.py

```diff
@@ -50,11 +50,31 @@ def test_anthropic_initialization() -> None:
     ChatAnthropic(model="test", anthropic_api_key="test")
 
 
-def test_formatting() -> None:
-    messages: List[BaseMessage] = [HumanMessage(content="Hello")]
+@pytest.mark.parametrize(
+    ("messages", "expected"),
+    [
+        ([HumanMessage(content="Hello")], "\n\nHuman: Hello\n\nAssistant:"),
+        (
+            [HumanMessage(content="Hello"), AIMessage(content="Answer:")],
+            "\n\nHuman: Hello\n\nAssistant: Answer:",
+        ),
+        (
+            [
+                SystemMessage(content="You're an assistant"),
+                HumanMessage(content="Hello"),
+                AIMessage(content="Answer:"),
+            ],
+            "You're an assistant\n\nHuman: Hello\n\nAssistant: Answer:",
+        ),
+        (
+            [
+                SystemMessage(content="You're an assistant"),
+                AIMessage(content="Answer:"),
+            ],
+            "You're an assistant\n\nAssistant: Answer:",
```

Comment:
I believe this test case would error out if you passed it to our API because there's no \n\nHuman: